backend/src/main/java/ai/verta/modeldb/configuration/AppConfigBeans.java

```diff
@@ -170,6 +167,15 @@ public Migration migration(MDBConfig mdbConfig) throws Exception {
     return migration;
   }
 
+  @Bean
+  public ModelDBHibernateUtil hibernateUtil(MDBConfig mdbConfig) {
```

Comment:
what is this `@Bean` definition used for?

Reply:
Nothing it just gives a surety about session factory is created. that bean is not used anywhere.

Reply:
this can be addressed as a separate change, but we should be sure to use the beans when we can